CMakeLists.txt

```diff
@@ -37,3 +37,4 @@ add_subdirectory(subsys)
 add_subdirectory(modules)
 add_subdirectory(drivers)
 add_subdirectory(tests)
+add_subdirectory(suit_provisioning)
```

Comment:
Actually this does not really seem to fit in this place. Other directories follow the standard zephyr directories structure. Is there no way we could add this in the `SUIT` subsystem?

Reply:
Moved under subsys/suit/provisioning